app/models/project.rb

```diff
@@ -55,8 +55,12 @@ def docker_repo
   # branch - The String name of the branch in question.
   #
   # Returns true if new releases should be created, false otherwise.
-  def create_releases_for_branch?(branch)
-    release_branch == branch
+  def create_releases_for_branch?(branch, service_type, service_name)
```

Comment:
might be better to rename this to `create_release?(branch, service_type, service_name)`